src/main/java/core/basesyntax/service/impl/ReportGeneratorServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGeneratorService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorServiceImpl implements ReportGeneratorService {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line before the class declaration to follow Java best practices.